### PR TITLE
created feature landing page"

### PR DESCRIPTION
**Pull request by kundajose**
Uploading landing page

### CODE REVIEW

**Comment by kundajose** [comment]
My website HTML CSS and form validation

File: script.js [hunk 1]
```diff
@@ -8,3 +8,34 @@ function openNav() {
 function closeNav() {
   document.querySelector(".top-nav").style.removeProperty('width');
 }
+
```
Comment (Eli250) [inline script.js]: Remove unnecessary empty lines.

**Review by Eli250 — CHANGES_REQUESTED** [review]
Remove  unnecessary lines and make sure that  you have only one commit message.

File: script.js (hunk 1 above)
Comment (kundajose) [reply script.js]: I have removed it

**Comment by kundajose** [comment]
Add log in validation ad removed unnecessary lines.

**Review by Eli250 — APPROVED** [review]

**Pull request merged**